pkmntypes.py

```diff
@@ -22,3 +22,28 @@ def __init__(self, **kwargs):
 
 class Pokemon():
 	pass
+
+class BattleContext():
+	"""This is a class that describes the point of view of a given Pokemon on the battlefield. It provides
+	enough information for a user to make an informed decision about what turn to make next.   
+	"""
+	# will add make the docstrings comply with the sphinx format once this class is looking good
+
+	def __init__(self):
+		self.battle = battle
+		self.pokemon = pokemon
+		self.team = team
+		self.targets = targets
+		self.allies = allies
+		self.opponents = opponents
```

Comment:
in this context, all the variables on the right side are not initialized

Reply:
To fix this, would I need to set it equal to `data['key']` and then get rid of the other function I wrote below? @dyc3

Reply:
yeah, basically